### PR TITLE
SC-8595 – add missing changes after testing

### PR DESCRIPTION
**Pull request by monaweitzenberg**
# Description

Add missing gender; delete language hint out of the language files and hbs; removed space in the code ; remove wrong list-element from the brb landing page

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-8595

## Changes

Add missing gender; delete language hint out of the language files and hbs; removed space in the code; remove wrong list-element from the brb landing page

## Screenshots of UI changes

image
image
image

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -255,7 +255,7 @@
 				"emailDeclarationOfConsentSubject": "Der letzte Schritt zur Aktivierung für die {{title}}",
 				"errorCreatingRegistrationLink": "Fehler beim Erstellen des Registrierungslinks. Bitte selbstständig Registrierungslink im Nutzerprofil generieren und weitergeben {{errMessage}}.",
 				"errorImportingUsers": "Fehler: {{errorMessage}}",
-				"failedToCreateUser": "Fehler beim Erstellen des Nutzers. {{error}}",
+				"failedToCreateUser": "Fehler beim Erstellen des Nutzers\/der Nutzerin. {{error}}",
```
Comment (sarahschaefer) [inline locales/de.json:258]: da würde ich eher sagen  der Nutzer:in

Comment (sarahschaefer) [inline locales/de.json:258]: da würde ich eher sagen  der Nutzer:in

File: locales/de.json [hunk 2]
```diff
@@ -454,27 +454,27 @@
 				"displayName": "Anzeige-Name",
 				"domainName": "Domain-Name",
 				"domainName_pathInLDAP": "Domain-Name (Pfad im LDAP)",
-				"fullPathClassUsers": "Nutzer:in der Klasse mit vollem Pfad",
+				"fullPathClassUsers": "Nutzer:innen der Klasse mit vollem Pfad",
 				"hide": "Ausblenden",
 				"noSchoolCloud": "keine HPI Schul-Cloud",
-				"noSchoolCloudTitle": "Dies ermöglicht es Ihnen, dass Nutzer:in aus Ihrem LDAP nicht in die HPI Schul-Cloud übertragen werden",
+				"noSchoolCloudTitle": "Dies ermöglicht es Ihnen, dass Nutzer:innen aus Ihrem LDAP nicht in die HPI Schul-Cloud übertragen werden",
```
Comment (sarahschaefer) [inline locales/de.json:460]: dir, deinem

Dies ermöglicht es dir, dass Nutzer:innen aus LDAP nicht in die HPI Schul-Cloud übertragen werden

File: locales/de.json [hunk 3]
```diff
@@ -1259,10 +1259,10 @@
...
-			"registrationInvalidAlreadyUsed": "Dieser/e Nutzer:in dem dieser Registrierungslink gehört, hat sich bzw. wurde bereits registriert.",
+			"registrationInvalidAlreadyUsed": "Dieser\/e Nutzer:in dem dieser Registrierungslink gehört, hat sich bzw. wurde bereits registriert.",
 			"registrationInvalidContactAdvice": "Bitte kontaktiere eine verantwortliche Lehrkraft oder Admin und frage diesbezüglich nach. Diese\/r kann weiterhelfen oder auch einen neuen Registrierungslink erstellen, falls nötig.",
-			"registrationInvalidManualConsent": "Eine Lehrkraft oder Admin hat für den entsprechende/n Nutzer:in bereits die manuelle Einverständniserklärung durchgeführt.",
-			"registrationInvalidOutdatedLink": "Der Registrierungslink ist sehr alt und nicht mehr korrekt mit dem/r entsprechenden/r Nutzer:in verknüpft."
+			"registrationInvalidManualConsent": "Eine Lehrkraft oder Admin hat für den entsprechende\/n Nutzer:in bereits die manuelle Einverständniserklärung durchgeführt.",
+			"registrationInvalidOutdatedLink": "Der Registrierungslink ist sehr alt und nicht mehr korrekt mit dem\/r entsprechenden\/r Nutzer:in verknüpft."
```
Comment (sarahschaefer) [inline locales/de.json:1265]: vielleicht umstellen: 

der/die Nutzer:in ist nicht mehr....

**Review by sarahschaefer — CHANGES_REQUESTED** [review]

**Review by murat-merdoglu-hpi — APPROVED** [review]

**Review by sarahschaefer — APPROVED** [review]

**Pull request merged**